public/app/features/alerting/AlertRuleList.tsx

```diff
@@ -98,9 +98,9 @@ export class AlertRuleListUnconnected extends PureComponent<Props> {
       <Page navId="alert-list">
         <Page.Contents isLoading={isLoading}>
           <div className="page-action-bar">
-            <div className="gf-form gf-form--grow">
+            <InlineField grow="true">
```

Comment:
As this is legacy alerting; and has been deprecated I'm hesitant to make any changes here – we don't usually have the bandwidth to test and vet these changes. 

The legacy alerting code will be removed in Grafana 11 so if the SASS styles will stick around for a while we might want to skip these changes.

Reply:
This is anything under `public/app/features/alerting/`, right?

Reply:
Correct, the new alerting lives under `public/app/features/alerting/unified` :)